docs/topics/multiplatform-mobile/multiplatform-mobile-understand-project-structure.md

```diff
@@ -3,33 +3,25 @@
 The purpose of the Kotlin Multiplatform Mobile technology is unifying the development of applications with common 
 logic for Android and iOS platforms. To make this possible, it uses a mobile-specific structure of
 [Kotlin Multiplatform](multiplatform.md) projects.
-This page describes the structure of a basic cross-platform mobile project. Note that this structure isn’t the only
-possible way to organize your project; however, we recommend it as a starting point.
 
-To view the complete structure of your mobile multiplatform project, switch the view from **Android** to **Project**.
-
-![Select the Project view](select-project-view.png){width=200}
+This page describes the structure and components of a basic cross-platform mobile project: shared module, Android app,
+and an iOS app.
 
-A basic Kotlin Mobile Multiplatform project consists of three components:
```

Comment:
I’d use the autogenerated module names, otherwise it reads a bit weird (I mean, the repetition of ‘Android application’ is a bit confusing here: <<Android application is a Kotlin module that builds into an Android application>>; the first one is a title, but it’s unclear whether we define it recursively or refer to a module name which is different). 

Also, I’d add comments on how android and iOS apps use the shared module.

Something like:

_shared_ module is a Kotlin module that contains common logic for both Android and iOS applications. It builds into both an Android library and an iOS framework. It uses [Gradle](https://branch-kmm-onboarding.kotlin-web-site.labs.jb.gg/docs/gradle.html) as a build system that helps you automate your building process.

_androidApp_ is a Kotlin module that builds into an Android application. It uses Gradle as a build system. It depends on and uses the _shared_ module as a regular Android library.

_iosApp_ is an Xcode project that builds into an iOS application. It depends on and uses the _shared_ module as an iOS framework. It might either use it as a regular framework or via CocoaPods dependency based on what we’ve chosen the previous step for “iOS framework distribution”. In this tutorial, it’s a regular framework dependency.